### ISSUE TITLE
CentOS 6 Support (or not).

### ISSUE THREAD

**Issue by cedarlug** [dgibbs64/linuxgsm#980]
Posting in relation to the [steam discussion](http://steamcommunity.com/groups/linuxgsm/discussions/0/360671583801990566/#c360671583802371540) regarding gmodserver install errors with the glibc fix on CentOS 6.8.

Here's what I see with CentOS 6 issues when verifying the process on a local image.  This relates to gmodserver, but likely generalizes to others.

In checking to see if the issue can be reproduced on a bare CentOS 6 install, one is first confronted with the omission of tmux in the CentOS 6 base repo.  The tmux dependency needs to be either manually compiled or pulled from another repo (along with libevent).

During the `./gmodserver auto-install` process, util-linux is mis-identified as not being installed (util-linux-ng-2.17.2-12.24.el6.x86_64  is installed).  The glibc dependencies were marked as being satisfied.  When tmux & libevent are installed from source, tmux is still reported as a missing dependency (not unexpected).

The installation otherwise completes without incident.

[Upon starting, ](http://pastebin.com/xNq0jaaz), the server dies with [glibc errors](http://pastebin.com/NApYWqvj). 

To recap:
-  tmux isn't available in the default repo for CentOS 6
-  the util-linux check misses util-linux-ng (already a bug report for this #933) in CentOS 6
-  glibc fix isn't working for CentOS 6.

**Comment by jaredballou** [comment]
glibc fix is hit or miss with Insurgency on Centos 6.7, I'm moving to using Docker to run everything in a container to sidestep the problem.

**Comment by dgibbs64** [comment]
These issues should now be resolved and gmod is working again in CentOS 6. Glibc fixes are working again 8a6d7b92b739ff5826a5320d5f3d29a496c3b352. I have also added a note on the website about installing epel. Tmux can now be built from source without check_deps.sh causing an issue cdcc6290e5661f956b54bb53da3340020c162420 and util-linux-ng is now used is CentOS 6. So should all be ok now. I will be merging in to master soon.
